lib/shared/api_database.dart

```diff
@@ -163,16 +163,25 @@ class ApiDatabase {
 
   Future<bool> verifyPassword(String password) async {
     try {
-      // verify password if empty but has master key
-      ApiDatabase db = Locator.instance<ApiDatabase>();
-      String key = await db.getKeychain();
-      var value = await _processIsolate(
+      bool isValidPasssword = await await _processIsolate(
         method: 'verifyPassword',
         params: {'password': password},
       );
-      if (value) {
+      if (isValidPasssword) {
         unlocked = true;
       }
+      return isValidPasssword;
+    } catch (e) {
+      return false;
+    }
+  }
+
+  Future<bool> verifyLogin(String password) async {
```

Comment:
Why do we need this function? Could we add a comment in `verifyLogin` and `verifyPassword` explaining what they do?